packages/mui-material/src/ButtonBase/ButtonBase.js

```diff
@@ -335,12 +335,12 @@ const ButtonBase = React.forwardRef(function ButtonBase(inProps, ref) {
     focusVisible,
   };
 
-  const classes = useUtilityClasses(ownerState);
+  const { root: rootClass, ...otherClasses } = useUtilityClasses(ownerState);
 
   return (
     <ButtonBaseRoot
       as={ComponentProp}
-      className={clsx(classes.root, className)}
+      className={clsx(rootClass, className)}
```

Comment:
@garronej this may even be inlined with the changes you have been doing. Can you double-check?

Reply:
@mnajdova Good job for connecting the dots.  
Yes this something that I had done at one point, I think it's the good call good call.
I rolled back though because appling this change to every comoponents of `@mui/material` make (at least) one unit test fail.  
I didn't want to introduce breaking changes so I aborted the mission.